frodo.js

```diff
@@ -2,35 +2,33 @@
  * Extras Plugin
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.